media-libs/libjpeg-turbo/libjpeg_turbo-1.5.3.recipe

```diff
@@ -0,0 +1,138 @@
+SUMMARY="MMX, SSE, and SSE2 SIMD accelerated JPEG library"
+DESCRIPTION="libjpeg-turbo is a JPEG image codec that uses SIMD instructions \
+(MMX, SSE2, AVX2, NEON, AltiVec) to accelerate baseline JPEG compression and \
+decompression on x86, x86-64, ARM, and PowerPC systems. On such systems, \
+libjpeg-turbo is generally 2-6x as fast as libjpeg, all else being equal. On \
+other types of systems, libjpeg-turbo can still outperform libjpeg by a \
+significant amount, by virtue of its highly-optimized Huffman coding routines.
+In many cases, the performance of libjpeg-turbo rivals that of proprietary \
+high-speed JPEG codecs.
+libjpeg-turbo implements both the traditional libjpeg API as well as the less \
+powerful but more straightforward TurboJPEG API. libjpeg-turbo also features \
+colorspace extensions that allow it to compress from/decompress to 32-bit and \
+big-endian pixel buffers (RGBX, XBGR, etc.), as well as a full-featured Java \
+interface.
+libjpeg-turbo was originally based on libjpeg/SIMD, an MMX-accelerated \
+derivative of libjpeg v6b developed by Miyasaka Masaru. The TigerVNC and \
+VirtualGL projects made numerous enhancements to the codec in 2009, and in \
+early 2010, libjpeg-turbo spun off into an independent project, with the goal \
+of making high-speed JPEG compression/decompression technology available to \
+a broader range of users and developers."
+HOMEPAGE="https://www.libjpeg-turbo.org/"
+COPYRIGHT="1991-2018 Thomas G. Lane, Guido Vollbeding"
+LICENSE="BSD (3-clause)
+	IJG"
+REVISION="1"
+SOURCE_URI="https://github.com/libjpeg-turbo/libjpeg-turbo/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="1a17020f859cb12711175a67eab5c71fc1904e04b587046218e36106e07eabde"
+SOURCE_DIR="libjpeg-turbo-$portVersion"
+PATCHES="libjpeg_turbo-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libjpegVersion="62.2.0"
+libjpegVersionCompat="$libjpegVersion compat >= ${libjpegVersion%%.*}"
+libturbojpegVersion="0.1.0"
+libturbojpegVersionCompat="$libturbojpegVersion compat >= ${libturbojpegVersion%%.*}"
+
+PROVIDES="
+	jpeg_turbo$secondaryArchSuffix = $portVersion
+	lib:libjpeg$secondaryArchSuffix = $libjpegVersion
+	lib:libturbojpeg$secondaryArchSuffix = $libturbojpegVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+CONFLICTS="
+	jpeg$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jpeg_turbo${secondaryArchSuffix}_devel = $portVersion
+	devel:libjpeg$secondaryArchSuffix = $libjpegVersion
+	devel:libturbojpeg$secondaryArchSuffix = $libturbojpegVersion
+	"
+REQUIRES_devel="
+	jpeg_turbo$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	jpeg${secondaryArchSuffix}_devel
```

Comment:
I'm not sure this is a good idea. I'm just adding this comment in case we change `CONFLICTS`.
```diff
CONFLICTS_devel="
-	jpeg${secondaryArchSuffix}_devel
+	lib:libjpeg$secondaryArchSuffix
+	lib:libturbojpeg$secondaryArchSuffix
 	"
```